toolchain/parse/context.h

```diff
@@ -97,10 +97,14 @@ class Context {
 
   // Adds a node to the parse tree that has no children (a leaf).
   auto AddLeafNode(NodeKind kind, Lex::TokenIndex token, bool has_error = false)
-      -> void;
+      -> void {
+    tree_->node_impls_.push_back(Tree::NodeImpl(kind, has_error, token));
+  }
 
   // Adds a node to the parse tree that has children.
-  auto AddNode(NodeKind kind, Lex::TokenIndex token, bool has_error) -> void;
+  auto AddNode(NodeKind kind, Lex::TokenIndex token, bool has_error) -> void {
+    tree_->node_impls_.push_back(Tree::NodeImpl(kind, has_error, token));
+  }
```

Comment:
These functions look identical -- am I missing something?

Reply:
No idea -- I just moved what I found to be inline.

Reply:
I think they became identical when we removed the tracking of child count. It might be interesting to consider whether we want to either unify them or add a CHECK that the right one is being called (but not in this PR).